templates/gc-audience-en.html

```diff
@@ -19,4 +19,4 @@
 	</div>
 </div>
 {% include components/gc-servinfo/2col.html %}
-{{>deptfeatures}}
+{% include components/gc-features/features.html %}
```

Comment:
Use include relative for content sample

Reply:
As discussed [here](https://github.com/wet-boew/GCWeb/pull/2089#issuecomment-1402109378), include_relative wil not work in this case.